core/common/src/main/java/alluxio/underfs/UnderFileSystemRegistry.java

```diff
@@ -111,7 +111,8 @@ public static UnderFileSystem create(String path, Map<String, String> ufsConf) {
     for (UnderFileSystemFactory factory : factories) {
       try {
         // Use the factory to create the actual client for the Under File System
-        return new UnderFileSystemWithLogging(factory.create(path, ufsConf));
+        return new UnderFileSystemWithLogging(
+            factory.create(path, new UnderFileSystemConfiguration(ufsConf)));
```

Comment:
Where is the line between `Map` and `UnderFileSystemConfiguration`, which class(es) should be responsible for converting the two?